### ISSUE TITLE
Add a rule to check against white list 

### ISSUE THREAD

**Issue by mscreenie** [siriusphp/validation#21]
Is there a way to check against a list of values inside an array. E.g.

$acceptedValues = array("value1","value2");

Only validate if user input is equal to what's inside an array

**Comment by adrianmiu** [comment]
You can use the 'InList' validator: https://github.com/siriusphp/validation/blob/master/docs/rules.md#array-validators
```php
$countryList = array('USA', 'Germany');
$validator->addRule('country', 'inlist', array('list' => $countryList), 'The country you provided is not valid');
```
If you need so that the user selects specific values you need to do
```php
$countryList = array('USA', 'Germany');
$validator->addRule('country', 'equal', array('value' => $countryList), 'The country you provided is not valid');
```

**Comment by adrianmiu** [comment]
The idea is that not all rules can be added using the shorthand notation. So you can't do something like

```php
$validator->addRule('country', 'required | inlist(list=USA,Germany)');
```

**Comment by mscreenie** [comment]
Excellent, 

Is there a way of specifying a label to a rule instead of repeating the label? I am repeating the label "After Timeout", can I specify it just once? I tried several methods but this is the only one that I can use that is passed to ErrorMessages properly.

'preference[*][notification][after][timeout]' => array(
   array('Number', null, null, 'After Timeout')
   array('maxlength(max=20)', null, null, 'After Timeout')
);

**Comment by adrianmiu** [comment]
Unfortunately at the moment, there is no such solution. It is true that it would be very useful.

**Comment by adrianmiu** [comment]
Closed by opening issue #22
